### DISCUSSION TITLE
Documentation for calculateP2G

### DISCUSSION THREAD

**Discussion by lmurrow** [xiaosaiyao/epiregulon#13] [General]
For calculateP2G, the default exp_assay is "logcounts" but in the tutorial at [https://xiaosaiyao.github.io/epiregulon.book/basic-workflow-with-multiassayexperiment.html#data-preparation] the input GeneExpressionMatrix seems to use normalized but not log transformed counts (based on the range/distribution of the input assay names normalizedCounts). I'd imagine this could make a difference for aggregation. Which do you recommend?

**Comment by TomVuod** [comment]
Hello @Imurrow, we recommend to use library size normalized counts. The name "logcounts" is maintained for backward compatibility and it originates from initial mislabeling of the dataset on which we tested `epiregulon`. Probably it will be changed in one of the next releases of the package.
